lib/RateLimiterRedis.js

```diff
@@ -29,6 +30,8 @@ class RateLimiterRedis extends RateLimiterStoreAbstract {
       this.client = opts.storeClient;
     }
 
+    this.rejectIfRedisNotReady = !!opts.rejectIfRedisNotReady;
```

Comment:
@dmozgovoi Almost there. It should be private property with name `_rejectIfRedisNotReady` (underscore in the beginning). I think it is not necessary to create getter and setter for this option.

Reply:
Done